### ISSUE TITLE
Trying to save form gives WordPress Critical error

### ISSUE THREAD

**Issue by hliljegren** [pods-framework/pods-gravity-forms#138]
## Issue Overview
I have a form on a WP multisite that is connected to a pod. When clicking "Send" I get "A critical error has occurred on your website" and no pod item has been created. If I log in as admin, everything works OK. I guess there is an underlying permission problem somewhere. 

## Expected Behavior
Even if there is a permission problem I still shouldn't get a critical error. I expect something that tells me what went wrong.

## Form Upload
<details>
```
{"0":{"title":"Mitt examinationstips","description":"","labelPlacement":"top_label","descriptionPlacement":"below","button":{"type":"text","text":"Skicka","imageUrl":""},"fields":[{"type":"post_title","id":6,"label":"Titel","adminLabel":"","isRequired":true,"size":"large","errorMessage":"","visibility":"visible","inputs":null,"formId":1,"description":"","allowsPrepopulate":false,"inputMask":false,"inputMaskValue":"","inputMaskIsCustom":false,"maxLength":"","inputType":"","labelPlacement":"","descriptionPlacement":"","subLabelPlacement":"","placeholder":"Titel f\u00f6r ditt tips","cssClass":"","inputName":"","noDuplicates":false,"defaultValue":"","choices":"","conditionalLogic":"","productField":"","multipleFiles":false,"maxFiles":"","calculationFormula":"","calculationRounding":"","enableCalculation":"","disableQuantity":false,"displayAllCategories":false,"useRichTextEditor":false,"fields":"","displayOnly":""},{"type":"textarea","id":2,"label":"Behov","adminLabel":"","isRequired":true,"size":"small","errorMessage":"","visibility":"visible","inputs":null,"formId":1,"description":"","allowsPrepopulate":false,"inputMask":false,"inputMaskValue":"","inputMaskIsCustom":false,"maxLength":"","inputType":"","labelPlacement":"","descriptionPlacement":"","subLabelPlacement":"","placeholder":"Beskriv kortfattat behovet som tipset handlar om","cssClass":"","inputName":"","noDuplicates":false,"defaultValue":"","choices":"","conditionalLogic":"","productField":"","form_id":"","useRichTextEditor":false,"multipleFiles":false,"maxFiles":"","calculationFormula":"","calculationRounding":"","enableCalculation":"","disableQuantity":false,"displayAllCategories":false,"checkboxLabel":"","fields":"","displayOnly":""},{"type":"textarea","id":3,"label":"Sammanhang","adminLabel":"","isRequired":false,"size":"small","errorMessage":"","visibility":"visible","inputs":null,"formId":1,"description":"","allowsPrepopulate":false,"inputMask":false,"inputMaskValue":"","inputMaskIsCustom":false,"maxLength":"","inputType":"","labelPlacement":"","descriptionPlacement":"","subLabelPlacement":"","placeholder":"I vilket sammanhang g\u00e4ller tipset?","cssClass":"","inputName":"","noDuplicates":false,"defaultValue":"","choices":"","conditionalLogic":"","productField":"","form_id":"","useRichTextEditor":false,"multipleFiles":false,"maxFiles":"","calculationFormula":"","calculationRounding":"","enableCalculation":"","disableQuantity":false,"displayAllCategories":false,"checkboxLabel":"","fields":"","displayOnly":""},{"type":"textarea","id":4,"label":"L\u00f6sning","adminLabel":"","isRequired":true,"size":"small","errorMessage":"","visibility":"visible","inputs":null,"formId":1,"description":"","allowsPrepopulate":false,"inputMask":false,"inputMaskValue":"","inputMaskIsCustom":false,"maxLength":"","inputType":"","labelPlacement":"","descriptionPlacement":"","subLabelPlacement":"","placeholder":"Vilket problem l\u00f6ser tipset?","cssClass":"","inputName":"","noDuplicates":false,"defaultValue":"","choices":"","conditionalLogic":"","productField":"","form_id":"","useRichTextEditor":false,"multipleFiles":false,"maxFiles":"","calculationFormula":"","calculationRounding":"","enableCalculation":"","disableQuantity":false,"displayAllCategories":false,"fields":"","displayOnly":""},{"type":"textarea","id":5,"label":"Konkreta steg","adminLabel":"","isRequired":true,"size":"large","errorMessage":"","visibility":"visible","inputs":null,"formId":1,"description":"Ge s\u00e5 konkreta steg-f\u00f6r-steg-beskrivningar som m\u00f6jligt","allowsPrepopulate":false,"inputMask":false,"inputMaskValue":"","inputMaskIsCustom":false,"maxLength":"","inputType":"","labelPlacement":"","descriptionPlacement":"","subLabelPlacement":"","placeholder":"Ge s\u00e5 konkreta steg-f\u00f6r-steg-beskrivningar som m\u00f6jligt","cssClass":"","inputName":"","noDuplicates":false,"defaultValue":"","choices":"","conditionalLogic":"","productField":"","form_id":"","useRichTextEditor":true,"multipleFiles":false,"maxFiles":"","calculationFormula":"","calculationRounding":"","enableCalculation":"","disableQuantity":false,"displayAllCategories":false,"fields":"","displayOnly":""},{"type":"textarea","id":7,"label":"Konsekvens","adminLabel":"","isRequired":false,"size":"medium","errorMessage":"","visibility":"visible","inputs":null,"formId":1,"description":"","allowsPrepopulate":false,"inputMask":false,"inputMaskValue":"","inputMaskIsCustom":false,"maxLength":"","inputType":"","labelPlacement":"","descriptionPlacement":"","subLabelPlacement":"","placeholder":"Konsekvenser eller t\u00e4nkbara problem med l\u00f6sningen","cssClass":"","inputName":"","noDuplicates":false,"defaultValue":"","choices":"","conditionalLogic":"","productField":"","form_id":"","useRichTextEditor":false,"multipleFiles":false,"maxFiles":"","calculationFormula":"","calculationRounding":"","enableCalculation":"","disableQuantity":false,"displayAllCategories":false,"fields":"","displayOnly":""},{"type":"textarea","id":8,"label":"Relaterat","adminLabel":"","isRequired":false,"size":"small","errorMessage":"","visibility":"visible","inputs":null,"formId":1,"description":"L\u00e4nkar till relaterat material. Anv\u00e4nd g\u00e4rna \"Infoga\/redigera l\u00e4nk\" ovan f\u00f6r att l\u00e4gga till en URL","allowsPrepopulate":false,"inputMask":false,"inputMaskValue":"","inputMaskIsCustom":false,"maxLength":"","inputType":"","labelPlacement":"","descriptionPlacement":"","subLabelPlacement":"","placeholder":"","cssClass":"","inputName":"","noDuplicates":false,"defaultValue":"","choices":"","conditionalLogic":"","productField":"","form_id":"","useRichTextEditor":true,"multipleFiles":false,"maxFiles":"","calculationFormula":"","calculationRounding":"","enableCalculation":"","disableQuantity":false,"displayAllCategories":false,"checkboxLabel":"","fields":"","displayOnly":""},{"type":"textarea","id":11,"label":"Vem ligger bakom?","adminLabel":"","isRequired":false,"size":"small","errorMessage":"","visibility":"visible","inputs":null,"formId":1,"description":"Skriv g\u00e4rna en person per rad","allowsPrepopulate":false,"inputMask":false,"inputMaskValue":"","inputMaskIsCustom":false,"maxLength":"","inputType":"","labelPlacement":"","descriptionPlacement":"","subLabelPlacement":"","placeholder":"Fyll i namn p\u00e5 alla er som varit med och t\u00e4nkt till ifall n\u00e5gon vill kontakta dig\/er f\u00f6r att veta mer","cssClass":"","inputName":"","noDuplicates":false,"defaultValue":"","choices":"","conditionalLogic":"","productField":"","form_id":"","useRichTextEditor":false,"multipleFiles":false,"maxFiles":"","calculationFormula":"","calculationRounding":"","enableCalculation":"","disableQuantity":false,"displayAllCategories":false,"fields":"","displayOnly":""},{"type":"post_custom_field","id":15,"label":"Kategori","adminLabel":"","isRequired":false,"size":"medium","errorMessage":"","visibility":"visible","inputs":[{"id":"15.1","label":"F\u00f6rsta valet","name":""},{"id":"15.2","label":"Andra valet","name":""},{"id":"15.3","label":"Tredje valet","name":""}],"inputType":"checkbox","formId":1,"description":"","allowsPrepopulate":false,"inputMask":false,"inputMaskValue":"","inputMaskIsCustom":false,"maxLength":"","labelPlacement":"","descriptionPlacement":"","subLabelPlacement":"","placeholder":"","cssClass":"","inputName":"","noDuplicates":false,"defaultValue":"","choices":[{"text":"Juridiskt","value":"5","isSelected":false},{"text":"Teknik","value":"4","isSelected":false},{"text":"Tentafr\u00e5gor","value":"3","isSelected":false}],"conditionalLogic":"","productField":"","enablePasswordInput":"","multipleFiles":false,"maxFiles":"","calculationFormula":"","calculationRounding":"","enableCalculation":"","disableQuantity":false,"displayAllCategories":false,"useRichTextEditor":false,"checkboxLabel":"","postCustomFieldName":"built_in_taxonomies_tipskategori","pods_populate_related_items":true,"fields":"","displayOnly":""}],"version":"2.4.17","id":1,"nextFieldId":16,"useCurrentUserAsAuthor":false,"postContentTemplateEnabled":false,"postTitleTemplateEnabled":false,"postTitleTemplate":"","postContentTemplate":"","lastPageButton":null,"pagination":null,"firstPageCssClass":null,"postAuthor":"43","postFormat":"0","subLabelPlacement":"below","cssClass":"","enableHoneypot":false,"enableAnimation":false,"save":{"enabled":false,"button":{"type":"link","text":"Spara och forts\u00e4tt senare"}},"limitEntries":false,"limitEntriesCount":"","limitEntriesPeriod":"","limitEntriesMessage":"","scheduleForm":false,"scheduleStart":"","scheduleStartHour":"","scheduleStartMinute":"","scheduleStartAmpm":"","scheduleEnd":"","scheduleEndHour":"","scheduleEndMinute":"","scheduleEndAmpm":"","schedulePendingMessage":"","scheduleMessage":"","requireLogin":false,"requireLoginMessage":"","postCategory":"1","postStatus":"draft","confirmations":[{"id":"5e81b794de75d","name":"Standardbekr\u00e4ftelse","isDefault":true,"type":"message","message":"Tack f\u00f6r att du kontaktade oss! Vi h\u00f6r av oss snart.","url":"","pageId":"","queryString":""},{"id":"5e81f41a75d6f","name":"Examinationstips","type":"message","message":"Tack f\u00f6r ditt tips! Vi kommer snart att granska det och l\u00e4gga ut det bland \u00f6vriga tips! Kom ih\u00e5g att du g\u00e4rna f\u00e5r komma tillbaka och ge oss fler tips!","disableAutoformat":false,"pageId":0,"url":"","queryString":"","isDefault":false,"conditionalLogic":{"actionType":"show","logicType":"all","rules":[{"fieldId":"6","operator":"is","value":""}]},"isActive":true}],"notifications":[{"id":"5e81b794db2d6","isActive":true,"to":"{admin_email}","name":"Administrat\u00f6rsnotifikation","event":"form_submission","toType":"email","subject":"Ny anm\u00e4lan fr\u00e5n {form_title}","message":"{all_fields}"}],"feeds":{"pods-gravity-forms":[{"id":"1","form_id":"1","is_active":"1","feed_order":"0","meta":{"feedName":"Nytt examinationstips","pod":"examinationstips","pod_fields_behov":"2","pod_fields_behov_custom":"2","pod_fields_sammanhang":"3","pod_fields_sammanhang_custom":"3","pod_fields_losning":"4","pod_fields_losning_custom":"4","pod_fields_konkreta_steg":"5","pod_fields_konkreta_steg_custom":"5","pod_fields_konsekvens":"7","pod_fields_konsekvens_custom":"7","pod_fields_relaterat":"8","pod_fields_relaterat_custom":"8","pod_fields_kontakt":"11","pod_fields_kontakt_custom":"","wp_object_fields_post_title":"6","wp_object_fields_post_title_custom":"","wp_object_fields_post_content":"2","wp_object_fields_post_content_custom":"","wp_object_fields_post_excerpt":"","wp_object_fields_post_excerpt_custom":"","wp_object_fields_post_author":"","wp_object_fields_post_author_custom":"","wp_object_fields_post_date":"","wp_object_fields_post_date_custom":"","wp_object_fields_post_name":"","wp_object_fields_post_name_custom":"","wp_object_fields_post_modified":"","wp_object_fields_post_modified_custom":"","wp_object_fields_post_parent":"","wp_object_fields_post_parent_custom":"","wp_object_fields_comments":"","wp_object_fields_comments_custom":"","wp_object_fields_tipskategori":"15","wp_object_fields_tipskategori_custom":"","wp_object_fields__thumbnail_id":"","wp_object_fields__thumbnail_id_custom":"","custom_fields":[{"key":"","value":"","custom_key":"","custom_value":""}],"update_pod_item":"0","enable_markdown":"0","enable_current_post":"0","enable_prepopulate":"1","delete_entry":"1","feed_condition_conditional_logic":"0","feed_condition_conditional_logic_object":[]},"addon_slug":"pods-gravity-forms","event_type":null}]}},"version":"2.4.17"}
```
</details>

## Pods Package Export

<details>
```
{"meta":{"version":"2.7.20","build":1587546802},"pods":{"490":{"id":490,"name":"examinationstips","label":"Examinationstips","description":"Tips för att göra digitala examinationer","type":"post_type","storage":"meta","object":"","alias":"","fields":{"behov":{"id":491,"name":"behov","label":"Behov","description":"Beskriv kortfattat behovet som tipset handlar om","help":"","class":"","type":"text","weight":0,"pick_object":"","pick_val":"","sister_id":"","required":"1","text_allow_shortcode":"0","text_allow_html":"0","text_allowed_html_tags":"strong em a ul ol li b i","text_max_length":"255","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","text_repeatable":"0","text_placeholder":""},"sammanhang":{"id":492,"name":"sammanhang","label":"Sammanhang","description":"I vilket sammanhang gäller tipset?","help":"","class":"","type":"paragraph","weight":1,"pick_object":"","pick_val":"","sister_id":"","required":"0","paragraph_allow_html":"1","paragraph_oembed":"0","paragraph_wptexturize":"1","paragraph_convert_chars":"1","paragraph_wpautop":"1","paragraph_allow_shortcode":"0","paragraph_allowed_html_tags":"strong em a ul ol li b i","paragraph_max_length":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","paragraph_repeatable":"0","paragraph_placeholder":""},"losning":{"id":493,"name":"losning","label":"Lösning","description":"Vilket problem löser vi?","help":"","class":"","type":"wysiwyg","weight":2,"pick_object":"","pick_val":"","sister_id":"","required":"1","wysiwyg_editor":"tinymce","wysiwyg_media_buttons":"1","wysiwyg_oembed":"0","wysiwyg_wptexturize":"1","wysiwyg_convert_chars":"1","wysiwyg_wpautop":"1","wysiwyg_allow_shortcode":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","wysiwyg_repeatable":"0","wysiwyg_allowed_html_tags":""},"konkreta_steg":{"id":494,"name":"konkreta_steg","label":"Konkreta steg","description":"Ge så konkreta steg-för-steg-beskrivningar som möjligt","help":"","class":"","type":"wysiwyg","weight":3,"pick_object":"","pick_val":"","sister_id":"","required":"1","wysiwyg_editor":"tinymce","wysiwyg_media_buttons":"1","wysiwyg_oembed":"0","wysiwyg_wptexturize":"1","wysiwyg_convert_chars":"1","wysiwyg_wpautop":"1","wysiwyg_allow_shortcode":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","wysiwyg_repeatable":"0","wysiwyg_allowed_html_tags":""},"konsekvens":{"id":495,"name":"konsekvens","label":"Konsekvens","description":"Konsekvenser eller tänkbara problem med lösningen","help":"","class":"","type":"paragraph","weight":4,"pick_object":"","pick_val":"","sister_id":"","required":"0","paragraph_allow_html":"1","paragraph_oembed":"0","paragraph_wptexturize":"1","paragraph_convert_chars":"1","paragraph_wpautop":"1","paragraph_allow_shortcode":"0","paragraph_allowed_html_tags":"strong em a ul ol li b i","paragraph_max_length":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","paragraph_repeatable":"0","paragraph_placeholder":""},"relaterat":{"id":496,"name":"relaterat","label":"Relaterat","description":"Länkar till relaterat material","help":"","class":"","type":"wysiwyg","weight":5,"pick_object":"","pick_val":"","sister_id":"","required":"0","wysiwyg_editor":"tinymce","wysiwyg_media_buttons":"1","wysiwyg_oembed":"0","wysiwyg_wptexturize":"1","wysiwyg_convert_chars":"1","wysiwyg_wpautop":"1","wysiwyg_allow_shortcode":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","wysiwyg_repeatable":"0","wysiwyg_allowed_html_tags":""},"kontakt":{"id":506,"name":"kontakt","label":"Kontakt","description":"Personer som varit med och skapat tipset","help":"","class":"","type":"paragraph","weight":6,"pick_object":"","pick_val":"","sister_id":"","required":"0","paragraph_allow_html":"1","paragraph_oembed":"0","paragraph_wptexturize":"1","paragraph_convert_chars":"1","paragraph_wpautop":"1","paragraph_allow_shortcode":"0","paragraph_allowed_html_tags":"strong em a ul ol li b i","paragraph_max_length":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","paragraph_repeatable":"0","paragraph_placeholder":""}},"show_in_menu":"1","label_singular":"Examinationstips","public":"1","show_ui":"1","supports_title":"1","supports_editor":"0","publicly_queryable":"1","exclude_from_search":"0","capability_type":"custom","capability_type_custom":"examinationstips","capability_type_extra":"1","has_archive":"1","hierarchical":"0","rewrite":"1","rewrite_with_front":"1","rewrite_feeds":"0","rewrite_pages":"1","query_var":"1","can_export":"1","default_status":"pending","supports_author":"0","supports_thumbnail":"0","supports_excerpt":"0","supports_trackbacks":"0","supports_custom_fields":"0","supports_comments":"0","supports_revisions":"0","supports_page_attributes":"0","supports_post_formats":"0","built_in_taxonomies_category":"0","built_in_taxonomies_link_category":"0","built_in_taxonomies_post_tag":"0","menu_position":"0","menu_icon":" dashicons-welcome-learn-more","show_in_nav_menus":"1","show_in_admin_bar":"1","pfat_enable":"1","pfat_run_outside_loop":"0","pfat_append_single":"replace","pfat_filter_single":"the_content","pfat_append_archive":"append","pfat_filter_archive":"the_content","rest_enable":"0","read_all":"0","write_all":"0","label_new_item":"Nytt examinationstips","built_in_taxonomies_tipskategori":"1","pfat_single":"Examinationstips single","pfat_archive":"Examinationstipslista"},"549":{"id":549,"name":"tipskategori","label":"Tipskategorier","description":"","type":"taxonomy","storage":"meta","object":"","alias":"","fields":[],"show_in_menu":"1","label_singular":"Tipskategori","public":"1","show_ui":"1","hierarchical":"1","built_in_post_types_examinationstips":"1","rewrite":"1","rewrite_with_front":"1","rewrite_hierarchical":"1","capability_type":"default","capability_type_custom":"tipskategori","query_var":"0","sort":"0","built_in_post_types_custom_css":"0","built_in_post_types_customize_changeset":"0","built_in_post_types_oembed_cache":"0","built_in_post_types_page":"0","built_in_post_types_post":"0","built_in_post_types_user_request":"0","built_in_post_types_wp_block":"0","built_in_post_types_attachment":"0","menu_location":"default","menu_position":"0","show_in_nav_menus":"1","show_tagcloud":"1","show_tagcloud_in_edit":"1","show_in_quick_edit":"1","show_admin_column":"0","pfat_enable":"0","pfat_run_outside_loop":"0","pfat_append_archive":"append","rest_enable":"0","read_all":"0","write_all":"0"}}}```
</details>
## WordPress Environment
<details>
```
WordPress Version: 5.3.2

PHP Version: 7.2.24

MySQL Version: 5.5.64

Server Software: Apache

Your User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1 Safari/605.1.15

Session Save Path: /var/lib/php/session

Session Save Path Exists: No

Session Save Path Writeable: No

Session Max Lifetime: 1440

Opcode Cache:

Apc: No
Memcached: No
OPcache: No
Redis: No
Object Cache:

APC: No
APCu: No
Memcache: No
Memcached: No
Redis: No
WPDB Prefix: solams_32_

WP Multisite Mode: Yes

WP Memory Limit: 64M

Current Memory Usage: 51,052M

Current Memory Usage (real): 18,250M

Pods Network-Wide Activated: Yes

Pods Install Location: /var/www/html/wp-content/plugins/pods/

Pods Tableless Mode Activated: No

Pods Light Mode Activated: No

Currently Active Theme: UnderStrap

Currently Active Plugins:

ALT Lab add script adder: 1.0
Cloner: 1.8.0
Display Posts: 3.0.2
Duplicate Page: 4.2
Easy Appointments: 2.12.3
Gravity Forms: 2.4.17
Multisite Custom CSS: 1.0
Pods - Custom Content Types and Fields: 2.7.20
Pods Gravity Forms Add-On: 1.4.3
SOLA custom Dashboard: 1.0
WP Cassify: 2.2.1
WP Pusher: 3.0.2
```
</details>

**Comment by Brian-Milnes** [comment]
I have a similar problem - PODS GF add-on causes Fatal PHP Error

I have a form submitted via Gravity Forms which has a Gravity Flow process associated. When trying to update with new values clicking on th Submit button produces the following error:

`Fatal error: Uncaught Error: Class 'GFEntryDetail' not found
in /home/customer/www/asd-cert.org/public_html/wp-content/plugins/pods-gravity-forms/includes/Pods_GF_Addon.php on line 992

Call stack:

Pods_GF_Addon::_check_admin_referer()
wp-includes/class-wp-hook.php:287
WP_Hook::apply_filters()
wp-includes/class-wp-hook.php:311
WP_Hook::do_action()
wp-includes/plugin.php:478
do_action()
wp-includes/pluggable.php:1151
check_admin_referer()
wp-content/plugins/gravityflow/includes/steps/class-step-user-input.php:389
Gravity_Flow_Step_User_Input::maybe_process_status_update()
wp-content/plugins/gravityflow/class-gravity-flow.php:5630
Gravity_Flow::inbox_page()
wp-content/plugins/gravityflow/class-gravity-flow.php:5520
Gravity_Flow::inbox()
wp-includes/class-wp-hook.php:287
WP_Hook::apply_filters()
wp-includes/class-wp-hook.php:311
WP_Hook::do_action()
wp-includes/plugin.php:478
do_action()
wp-admin/admin.php:259`

**Comment by sc0ttkclark** [comment]
A quick update here -- I'm still going through testing and fixes for compatibility issues with the latest GF major release. As soon as that finishes I will push this all up as a new major update for Pods GF for everyone to get.

**Comment by sc0ttkclark** [comment]
To the OP, I haven't reproduced your issue yet but I will follow up with it before I release the other fixes.

**Comment by sc0ttkclark** [comment]
Fixed in Pods GF 1.4.5
